content/en/users/check-in/faq/_index.md

```diff
@@ -55,7 +55,7 @@ handle
 [Scoping elements](https://docs.microsoft.com/en-za/azure/active-directory/develop/active-directory-single-sign-on-protocol-reference#scoping>).
 Check-in can be configured to omit the scoping element from the authentication
 requests sent to such IdPs in order to allow successful logins. Please contact
-the CheckIn support team and include a screenshot of your error.
+the Check-in support team and include a screenshot of your error.
 
```

Comment:
Please, write here the Check-in support address.

Reply:
The email address has been added

Reply:
I would use `check-in@egi.eu`, it's simpler/cleaner/easier to retain.

Reply:
I have re-edited the ML address. We inform the end users to contact us via `checkin-support@mailman.egi.eu` (instead of check-in@egi.eu) because this mailing list creates automatically GGUS ticket which help us to track the issues of Check-in.

Reply:
Perfect, thanks!